pydata/api.py

```diff
@@ -118,14 +119,19 @@ class SponsorshipAPIClient(BaseAPIClient):
     model = Sponsorship
 
     def parse(self, sponsor):
+        domain = urlparse(sponsor['external_url']).netloc
+        try:
+            organization = Organization.objects.filter(
+                Q(fullname=sponsor['name']) | Q(domain=domain)
+            ).first()
```

Comment:
`.first()` returns the first object matched by the queryset, **or None if there is no matching object**.